### ISSUE TITLE
GBM Importer

### ISSUE THREAD

**Issue by moyanojv** [iamDecode/sklearn-pmml-model#20]
Hello, 

Are in your scope an importer for GBM models?

Thanks in advance

**Comment by iamDecode** [comment]
Yep, I would definitely like to include GBM models. What language/toolkit do you use to train and export the GBM model to PMML?

**Comment by moyanojv** [comment]
In R we use to train 'gbm' and/or 'LightGBM'. The transformation to PMML is done using 'pmml' and/or 'r2pmml'

Thanks
